internal/packages/packagecontent/topackage.go

```diff
@@ -83,3 +91,49 @@ func PackageFromFiles(ctx context.Context, scheme *runtime.Scheme, files Files)
 
 	return
 }
+
+func processManifestFile(ctx context.Context, scheme *runtime.Scheme, manifest *manifestsv1alpha1.PackageManifest, path string, content []byte) (*manifestsv1alpha1.PackageManifest, error) {
+	if manifest != nil {
+		return manifest, packages.ViolationError{
```

Comment:
After further analysis, to me it looks like this code branch is never going to happen because there's only one possible path for the manifest file.
@eqrx am I correct or am I missing anything?
If that's true, I'd prefer to remove the whole "if".

Reply:
ok I was wrong...my bad...I added the test in `topackage_test.go` now 👍